### PR TITLE
Refine Pixel increment in file information

### LINKED ISSUE #11: [file info] pixel increment
The entry "Pixel increment" in the file info now adopts the unit defined in the header. To have a better UX, please transform it to user friendly units, like milliarcsec, arcsec, arcmin, or degree, depending on the value itself. If it is velocity, please use km/s.

### PR DESCRIPTION
**Pull request by duidae**
fix #11

### CODE REVIEW

File: FileInfoLoader.cc [hunk 1]
```diff
@@ -443,11 +446,13 @@ bool FileInfoLoader::fillHdf5ExtFileInfo(FileInfoExtended* extendedInfo, string&
         crDegStr = fmt::format("[{}, {}]", cr1, cr2);
         if (!(cdelt1 == 0.0 && cdelt2 == 0.0))
             axisInc = fmt::format("{}, {}", unitConversion(cdelt1, cunit1), unitConversion(cdelt2, cunit2));
+        if (!(bmaj == 0.0 && bmin == 0.0 && bpa == 0.0))
```
Comment (veggiesaurus) [inline FileInfoLoader.cc]: if either bmaj or bmin are zero, we don't really have beam info, surely?

Comment (kswang1029) [reply FileInfoLoader.cc]: none of them should be zero, otherwise a beam cannot be defined.

Comment (veggiesaurus) [reply FileInfoLoader.cc]: bpa can be zero, right? 

the current check only guards against all three of them being zero.

Comment (kswang1029) [reply FileInfoLoader.cc]: Oops, right. None of bmaj and bmin should be zero. bpa can be zero.

**Pull request merged**